scanreader/scans.py

```diff
@@ -671,6 +672,114 @@ class Scan2020(Scan5Point3):
     """ ScanImage 2020"""
     pass
 
+class Scan2021(Scan5Point3):
+    """ ScanImage 2021"""
+
+    # New meta data that gets saved into the header when Mini2p unwarping
+    # toolbox was used to correct the data 
+    # 
+    # SI.Custom.MINI2P.MINI2P_Corrected | 'true' or '1' 
+    # SI.Custom.MINI2P.tf_path | str | transformMatrixDirectory
+    # SI.Custom.MINI2P.tfused | str | TransformMatrix_used
+    # SI.Custom.MINI2P.MINI2P_system | str | 
+    # SI.Custom.MINI2P.MINI2P_scope | str |
+    # SI.Custom.MINI2P.MINI2P_objective | str | 
+
+    # Similar entries will be created on the level of the MDF 
+    # transformMatrixDirectory
+    # system
+    # scope
+    # objective
+
+    # @property
```

Comment:
Move the documentation comments to the main docstring

Remove the commented out code, as you've replaced it further down.

Reply:
Done